assets/js/src/newsletters/types.jsx

```diff
@@ -245,9 +250,11 @@ class NewsletterTypes extends React.Component {
     return (
       <div>
         <link rel="prefetch" href={window.mailpoet_editor_javascript_url} as="script" />
-        <h1>{MailPoet.I18n.t(window.mailpoet_newsletters_count === 0 ? 'createFirstEmailTitle' : 'pickCampaignType')}</h1>
 
-        <Breadcrumb step="type" />
+        {this.props.showHeader
+          && <h1>{MailPoet.I18n.t(window.mailpoet_newsletters_count === 0 ? 'createFirstEmailTitle' : 'pickCampaignType')}</h1>
```

Comment:
The header is never shown because only the last part of the logical expression is returned.

Reply:
Good catch, fixed and also rebased since there were conflicts